src/__tests__/__snapshots__/openapi.test.js.snap

```diff
@@ -93,7 +94,7 @@ Object {
         },
         "items": Object {
           "items": Object {
-            "$ref": "#/definitions/Foo",
+            "$ref": "#Foo",
```

Comment:
This is not really the correct `$ref` for `Swagger`, should be fixed in next PR.